fbw-a380x/src/systems/instruments/src/PFD/FMA.tsx

```diff
@@ -1363,6 +1411,38 @@ class BC3Cell extends DisplayComponent<{ isAttExcessive: Subscribable<boolean>;
 
   private tdReached = false;
 
+  private isUnrestrictedClimbDescent = 0;
```

Comment:
Didn't look at the full file, is there some way where you don't need the whole code copied here again?

Reply:
Not sure about that. The `FMA` class needs to call `handleFMABorders` for some reason when changing anything connected with `getBC3Message`, so it needs basically the same subs as the `BC3Cell` class. I don't know whether there's a way to make the code "shared", so I rather kept it "simple" and did it the same way as for the other parameters passed into `getBC3Message`, which were there before (put the same code into both classes).

Thank you for the review!

Reply:
You could pass it as a prop from the FMA component all the way to the BC3 cell. You can see #9606 for some inspiration

Reply:
> You could pass it as a prop from the FMA component all the way to the BC3 cell. You can see #9606 for some inspiration

Okay, I'll check it out, thanks!

Reply:
I've done it the same way as another message is in #9606. Done!